homeassistant/components/plant/const.py

```diff
@@ -0,0 +1,42 @@
+"""Const for Plant."""
```

Comment:
Please 🙏 move existing constants in a preliminary PR.

Reply:
Please check out #91008